src/ops.rs

```diff
@@ -1111,7 +1117,8 @@ fn op_listen(
   data: &'static mut [u8],
 ) -> Box<Op> {
   assert_eq!(data.len(), 0);
-  if !state.flags.allow_net {
+  // FIXME
+  if state.flags.permissions_check_net("listen").is_err() {
```

Comment:
"listen" isn't a domain.

I think it would be good if `permissions_check_net()` took the Op's name and displayed it along with an optional domain. 

You can get the op name with this: `msg::enum_name_any(base.inner_type())`

Reply:
True! (That was the "FIXME".)

I think there is a larger question of if you allow net access for "listen" does that really mean you want net access to `www.facebook.com`, the message would be a little misleading if that's the case. Will check this out.

Reply:
I haven't addressed this yet.